config-linux.md

```diff
@@ -337,14 +337,19 @@ The following parameters can be specified to set up the controller:
 
 * **`weight`** *(uint16, OPTIONAL)* - specifies per-cgroup weight. This is default weight of the group on all devices until and unless overridden by per-device rules.
 * **`leafWeight`** *(uint16, OPTIONAL)* - equivalents of `weight` for the purpose of deciding how much weight tasks in the given cgroup has while competing with the cgroup's child cgroups.
-* **`weightDevice`** *(array of objects, OPTIONAL)* - specifies the list of devices which will be bandwidth rate limited. The following parameters can be specified per-device:
+* **`weightDevice`** *(array of objects, OPTIONAL)* - an array of per-device bandwidth weights. Each entry has the following structure:
```

Comment:
The “Each entry…” sentence [should be on it's own line][1] like it is [here][2].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc6/style.md#one-sentence-per-line
[2]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc6/config.md#L165